pypresence/client.py

```diff
@@ -23,13 +42,24 @@ def register_event(self, event: str, func: callable, args: dict = {}):
         self._events[event.lower()] = func
 
     def unregister_event(self, event: str, args: dict = {}):
+        """
+        Unhook an event from a function. Will auto unsubscribe from the event as well.
+        :param event: (str) the event to unhook
+        :param args: (dict) optional args used in unsubscription
+        :return: pypresence.Response
+        """
         event = event.lower()
         if event not in self._events:
             raise EventNotFound
         self.unsubscribe(event, args)
         del self._events[event]
 
     def on_event(self, data):
+        """
+        TODO
```

Comment:
You forgot to finish this doc?

Reply:
I have no idea what the function does so it can be done by someone else

Reply:
Undocumented function https://qwertyquerty.github.io/pypresence/html/doc/client.html.
Could've prepended a _ to it if it was a private function.

Reply:
Ok lol, doesn't really need a doc shouldn't be called by anyone manually. Could do with an underscore prefix.